nextcloud: 27.0.1 -> 27.0.2, 26.0.4 -> 26.0.5, 25.0.9 -> 25.0.10; update apps

Ma27 opened the pull request:
## Description of changes

ChangeLog: https://nextcloud.com/changelog/#27-0-2
## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

Ma27 committed:
nextcloud: 27.0.1 -> 27.0.2

ChangeLog: https://nextcloud.com/changelog/#27-0-2

Ma27 committed:
nextcloudPackages*: update

Ma27 committed:
nextcloud25: 25.0.9 -> 25.0.10

ChangeLog: https://nextcloud.com/changelog/#25-0-10

Ma27 committed:
nextcloud26: 26.0.4 -> 26.0.5

ChangeLog: https://nextcloud.com/changelog/#26-0-4

matthiasbeyer commented:
Result of `nixpkgs-review pr 249123` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>nextcloud25</li>
    <li>nextcloud26</li>
    <li>nextcloud27</li>
  </ul>
</details>

bachp reviewed (approved):
26.0.5 tested and working

dotlambda commented on pkgs/servers/nextcloud/packages/26.json:
```diff
@@ -110,18 +110,18 @@
-    "sha256": "12dg1bklv2jhmj5dnz4ram6zvgf8kipfz77g1lcn77fyhzqw6y1z",
-    "url": "https://github.com/nextcloud/maps/releases/download/v1.1.0/maps-1.1.0.tar.gz",
+    "sha256": "0517kakkk7lr7ays6rrnl276709kcm5yvkp8g6cwjnfih7pmnkn9",
+    "url": "https://github.com/nextcloud/maps/releases/download/v1.1.0-2a-nightly/maps-1.1.0-2a-nightly.tar.gz",
```
This doesn't look right.

Ma27 replied:
nc4nix actually detects pre-releases, but apparently it doesn't cope well with whatever this gibberish is :shrug: 

will revert the relevant parts.

bachp reviewed (approved):
27.0.2 is working too

Ma27 committed:
nextcloudPackages*: pin `maps` to stable 1.1.0

No idea what this gibberish disguised as tag is, but nc4nix doesn't seem
to cope well with it. For now, let's pin `maps` to the stable 1.1 release (as
it's the case for nextcloud27 already) since 1.1 is supported for all of
v25 to v27[1], so this seems reasonable to do.

[1] https://github.com/nextcloud/maps/blob/v1.1.0/appinfo/info.xml#L36

Ma27 commented:
Taking care of the backport now.

Ma27 commented:
-> #249723
